airlock/settings.py

```diff
@@ -63,6 +63,15 @@ def get_env_var(name):
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = get_env_var("DJANGO_DEBUG") == "True"
 
+if DEBUG:  # pragma: no cover
+    DEBUG_TOOLBAR = os.environ.get("DEBUG_TOOLBAR", "false").lower() in [
+        "1",
+        "true",
+        "yes",
```

Comment:
I'm not wild about having an ad-hoc list of truthy alternatives for this one specific value. If we want to have multiple ways to spell booleans then I think we should have a function somewhere which handles these consistently, and rejects unrecognised values.

Again, I realise this is a dev only setting and so it doesn't matter that much how we handle things here. But as soon as code gets merged it starts to gain the status of an established pattern that then gets copied elsewhere.